api/src/test/resources/log4j.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+
+    This Source Code Form is subject to the terms of the Mozilla Public License,
+    v. 2.0. If a copy of the MPL was not distributed with this file, You can
+    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+
```

Comment:
@dkayiwa  when I try to run the test cases with out this file I get the warning as follows.
```
Running org.openmrs.module.dhisreport.api.db.hibernate.HibernateDHIS2ReportingDAOTest
log4j:WARN No appenders could be found for logger (org.springframework.test.context.junit4.SpringJUnit4ClassRunner).
log4j:WARN Please initialize the log4j system properly.
```
This file is added in other PR also because there it was helping in logging properly and avoiding unnecessary logs.